### ISSUE TITLE
Review Sitemap

### ISSUE THREAD

**Issue by Maroyafaied** [DOI-ONRR/onrr.gov-site#2584]
@AlexandracmONRR please give each person a section to review. Thanks

**Comment by AlexandracmONRR** [comment]
Thank you all for helping me to review the onrr.gov sitemap update!

For review: [onrr.gov sitemap](https://cp63dj.axshare.com/?id=jl5r59&g=14)

**Erin**: Primary Navigation, Home (0.0), Getting Started (1.0)
**Christine**: Reporting (2.0), References (3.0)
**Lindsay**: Paying (4.0), Compliance and Enforcement (5.0), Indian Resources (6.0)
**Maroya**: About ONRR (7.0), Header Links (8.0), Footer (9.0)

**Comment by eelzi-ONRR** [comment]
@AlexandracmONRR I finished reviewing my assigned sections. Let me know if you have any questions about the comments I made

Overall it's very thorough and easy to follow!

**Comment by cthomasONRR** [comment]
@AlexandracmONRR I finished reviewing my sections of the sitemap. I'm sorry for the amount of comments I added! We can set up a meeting with @Maroyafaied to go through them all if that's easier. Overall, it's really good, but I was struggling with inconsistencies between sitemap pages and found a lot of internal and external links, and individual file downloads that are missing. I tried to note everything via Axure, hence all the comments lol.
